### ISSUE TITLE
[Blue Burst] Retrieving a stock baby mag from bank AND another item ends up with both items not appearing in inventory

### ISSUE THREAD

**Issue by nolrinale** [fuzziqersoftware/newserv#478]
**Describe the bug**
While playing the quest Warriors Pride (EP4 Solo) I attempted to take out a few items from my bank mid quest, and end up losing them all after withdrawing them they never appeared in my inventory.

**To reproduce**
Fill in steps to reproduce the behavior, such as:
1. Create a solo EP4 room and load the quest Warrior's Pride with ASH and BERNIE with you
2. Kill a few monsters then port back to P2, go to your bank and retrieve any item I tested with Weapons, Baby Mags, Photon Drops , Units
3. The items won't appear in your inventory, and they wont appear in the bank anymore because you just retrieved them
4. Only the Unit for some reason was able to be withdraw succesfully, everything else was lost.

**To reproduce ROUND 2**
Fill in steps to reproduce the behavior, such as:
1. retrieve a stock baby mag deposited earlier AND another item a photon drop or a sol atomizer for example
2. both items wont appear in the inventory and they are effectively lost
3. make sure to do these steps without closing the bank withdraw window, withdraw the mag first, then something else afterwards then close the withdraw window and check your inventory
4. after a while I also ended up getting Error 100 attached to the extended 2nd log

NOTE: I had previously farmed most of those baby mags in the mines so they all come from Containers, not from brand new characters.

**Game version(s) (choose one or more of the following):**
BB

**Additional context**
Session Log
[BankItemLostQuestEP4.txt](https://github.com/fuzziqersoftware/newserv/files/15202314/BankItemLostQuestEP4.txt)

Extended Session Log w Error 100
[BankItemLostQuestEP4Error100.txt](https://github.com/fuzziqersoftware/newserv/files/15202570/BankItemLostQuestEP4Error100.txt)

```
[193463]: 0010 | 00 00 00 00 00 01 FF FF                         |
[193463]: W 193463 2024-05-03 16:21:19 - [Server] Error processing client command: item not present
[193463]: I 193463 2024-05-03 16:21:19 - [Server] Client disconnected: C-11 on fd 54

```

The server console says I got them but in-game I don't have them
![IMG_2477](https://github.com/fuzziqersoftware/newserv/assets/5244595/31fca00b-c760-4e18-8d33-76e8a96d3e56)

**Comment by nolrinale** [comment]
More mag related stuff here
https://github.com/fuzziqersoftware/newserv/issues/456

**Comment by fuzziqersoftware** [comment]
This is because the NPCs consume some item ID slots from the creating player at the time the NPCs are created, so items created by the server after that conflict with the NPCs' items. This should be fixed by e42cfb64.

**Comment by nolrinale** [comment]
My second diagnostic was completely wrong, I checked it again and its working as intended, closing this one
